bin/storm

```diff
@@ -88,12 +88,12 @@ def get_config_opts():
     global CONFIG_OPTS
     return "-Dstorm.options=" + ','.join(map(quote_plus,CONFIG_OPTS))
 
-if not os.path.exists(STORM_DIR + "/RELEASE"):
+if not os.path.exists(STORM_DIR + "/lib/"):
```

Comment:
Please use `STORM_LIB_DIR` instead of `STORM_DIR + "/lib/"`.

Also the variable `STORM_RELEASE_DIR` is no longer used in the file and should be deleted.